src/components/button/button.stories.mdx

```diff
@@ -64,6 +64,16 @@ Primary buttons can be `fullWidth`.
   </Story>
 </Preview>
 
+Primary buttons can be set to `noWrap` mode to prevent text wrapping while in flex container.
+<Preview>
+  <Story name="primary/noWrap" parameters={{ info: { disable: true }}}>
```

Comment:
This `parameters={{ info: { disable: true }}}` means that chromatic won't take a snapshot of these, I think there's value of having a Chromatic snapshot here. Just for one of them though, there's no need to have it on all of them because it's the same functionality just with the different button styling

Reply:
@sjohnson-sage Are you sure about this one? If removed, "ugly" PropTypes table and story source code is visible on docs page.

Reply:
Yes I'm sure. Can you show me what you mean by the prop types table please? Just removing the `parameters` here shouldn't affect the prop types table or source code display at all.